README.md

```
@@ -0,0 +1,143 @@
+
+Java Development Kit 11+
+
+Apache Maven 3.6.0+
+
+Git 2.24+
+
+##Task
+
+Tasks 1-17 weight: 40 points
+
+##### 1. Create maven project with 5 modules (see Java 9 modules) :
```

Markdownlint

Expected: h3; Actual: h5
Heading levels should only increment by one level at a time

(MD001, heading-increment)